tests/foreman/api/test_role.py

```diff
@@ -186,24 +153,28 @@ def user_config(self, user):
         :param user: The nailgun.entities.User object of an user with passwd
             parameter
         """
-        return ServerConfig(auth=(user.login, user.passwd), url=self.sat_url, verify=False)
+        return ServerConfig(
+            auth=(user.login, user.passwd), url=f'https://{settings.server.hostname}', verify=False
+        )
 
-    @classmethod
-    def setUpClass(cls):
-        """Creates two orgs and locations that will be used by all the
-        underlying tests
-        """
-        super().setUpClass()
-        cls.sat_url = f'https://{settings.server.hostname}'
-        # These two will be used as role taxonomies
-        cls.role_org = entities.Organization().create()
-        cls.role_loc = entities.Location().create()
-        # These two will be used as filter taxonomies
-        cls.filter_org = entities.Organization().create()
-        cls.filter_loc = entities.Location().create()
+    @pytest.fixture
+    def role_taxonomies(self):
+        """Create role taxonomies"""
+        return {
+            'org': entities.Organization().create(),
+            'loc': entities.Location().create(),
+        }
+
+    @pytest.fixture
+    def filter_taxonomies(self):
+        """Create filter taxonomies"""
+        return {
+            'org': entities.Organization().create(),
+            'loc': entities.Location().create(),
+        }
```

Comment:
Why we creating similar object fixtures here ? can't we keep same fixture for both  `role_taxonomies` and `filter_taxonomies` with some common name as 'taxonomies' uses based on the scope of fixture ?

Reply:
seeing below code I can understand there are times you require both fixture simultaneously. ignore it